src/cps/loader.hpp

```diff
@@ -140,6 +140,8 @@ namespace cps::loader {
         version::Schema version_schema;
     };
 
+    constexpr static std::string_view CPS_VERSION = "0.12.0";
```

Comment:
`constexpr inline` for a variable. Don't want a static in every TU that includes the header.